live-examples/html-examples/inline-text-semantics/bdo.html

```diff
@@ -0,0 +1,7 @@
+<h1>Famous seaside songs</h1>
+
+<p>The English song "Oh I do like to be beside the seaside"</p>
+
+<p>Looks like this in Hebrew: <span dir="rtl">אה, אני אוהב להיות ליד חוף הים</span></p>
+
+<p>In the computer's memory, this is stored as <bdo>אה, אני אוהב להיות ליד חוף הים</bdo></p>
```

Comment:
According to https://www.w3.org/International/articles/inline-bidi-markup/#override, `dir` is mandatory for `<bdo>`, and the example accompanying that article uses it: https://www.w3.org/International/articles/inline-bidi-markup/index-data/bdo.html.

(Our docs should say this, too.)